can_tools/scrapers/official/federal/CDC/cdc_testing_cases.py

```diff
@@ -39,15 +39,15 @@ class CDCHistoricalTestingDataset(FederalDashboard, ETagCacheMixin):
     # Send URL and filename that Mixin will use to check the etag
     def __init__(self, execution_dt: pd.Timestamp = pd.Timestamp.utcnow()):
         ETagCacheMixin.initialize_cache(
-            self, cache_url=self.fetch_url, cache_file="cdc_historical_testing.txt"
+            self, cache_url=self.fetch_url, cache_file=self.cache_file
         )
         super().__init__(execution_dt=execution_dt)
 
     def fetch(self) -> pd.DataFrame:
         # select only the columns we care about in order to speed up query
         data = pd.read_json(
             f"{self.fetch_url}?$limit={SODA_API_RESPONSE_LIMIT}"
-            "&$select=fips_code,date,cases_per_100k_7_day_count,percent_test_results_reported"
+            f"&$select=fips_code,{self.date_column},cases_per_100k_7_day_count,percent_test_results_reported"
```

Comment:
The two datasets have different names for the data column, so we have to specify

Reply:
LOL, of course they do. 🙄